backend/clubs/views.py

```diff
@@ -173,6 +178,20 @@ def filter_note_permission(queryset, club, user):
     return queryset
 
 
+def hour_to_string_helper(hour):
+    hour_string = ""
+    if hour == 0:
+        hour_string = "12am"
+    elif hour < 12:
+        hour_string = str(hour) + "am"
```

Comment:
It is generally good practice to do `f"{hour}am"` or `"{}am".format(hour)` instead of `str(hour) + "am"`.